frontend-react/src/components/StaticAlert.tsx

```diff
@@ -17,13 +17,16 @@ export const StaticAlert = ({
     const alertClasses = classNames({
         "usa-alert": true,
         "usa-alert--success": type === "success",
+        "usa-alert--success usa-alert--slim": type === "success slim",
```

Comment:
[nit] `usa-alert--slim` could be handled independently

```suggestion
        "usa-alert--slim": type.indexOf("slim") > -1,
```